### ISSUE TITLE
Disc 4 problem 10)

### ISSUE THREAD

**Issue by sxysun-zz** [dsherma7/Math.UCDavis#2]
I think the solution is somewaht not right, and why would there be a taylor remainder formula included in such problem?

**Comment by dsherma7** [comment]
Yeah I meant to use the Taylor series definition, not remainder theorem. Thanks for pointing this out!

**Comment by sxysun-zz** [comment]
yeah that is right, but I still have questions. the problem told us to find the taylor series centered at x=-1 not x=0, then how should we transform this equation to do that?

**Comment by sxysun-zz** [comment]
ok i got it, use u = x+1 thanks for answering!

**Comment by dsherma7** [comment]
Yeah you can't just sub in x+1 because the derivative terms on the Taylor series definition change when we change the center. I re-did my answer and tried to go over this some more.

**Comment by sxysun-zz** [comment]
ok but why did I got the exactly same answers by just substituting x+1?
* I mean substituting this will change the `a` right, and that is exactly what we want, besides the series should start at `-1/4` instead of `3/4` as written on the answer key

**Comment by dsherma7** [comment]
Where did you substitute x+1 in?

**Comment by sxysun-zz** [comment]
![img_20171107_211331](https://user-images.githubusercontent.com/29686349/32532721-1ccc59c8-c401-11e7-913b-460efec029f4.jpg)

**Comment by dsherma7** [comment]
That’s clever. So, if you start from the series definition of a function, you can’t just substitute x+1 without adjusting the nth derivative term (as I wrote in the key). In your method, because the Taylor series of f(x) at a=-1 should match the Taylor series of f(x-(-1)) centered at a=0 you can just sub in right away. I’m going to take this for the key. ha

Thanks!
